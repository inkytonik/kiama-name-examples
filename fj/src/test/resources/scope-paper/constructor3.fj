// class/constructors.spt:nullary constructor
class A extends Object {
  A() {
    super();
  }
}
new A()
