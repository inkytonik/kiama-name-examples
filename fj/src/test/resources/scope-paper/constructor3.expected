src/test/resources/scope-paper/constructor3.fj:7:1:info: new A(): A
new A()
^
A
